app/components/work_packages/types/pattern_autocompleter.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+# -- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2024 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+# ++
+
+module WorkPackages
+  module Types
+    class PatternAutocompleter < Primer::Forms::BaseComponent
+      delegate :name, to: :@input
+
+      def initialize(input:, value:, suggestions:)
+        super()
+        @input = input
+        @value = value
+        @suggestions = suggestions
+      end
+
+      def select_item_action
+        {
+          action: "click->pattern-autocompleter#suggestions_select",
```

Comment:
I just spotted this one and noticed that I didn't try using the mouse to select an autocomplete entry. For me it doesn't work to click on an autoselect entry. The pop-up closes, but nothing gets selected.

Reply:
New learning: Selecting with the mouse works after opening the suggestion list via the arrow down button, but not when using the mouse after starting autocompletion via typing.